CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributor's Guide
+
+HOW TO CONTRIBUTE TO OPEN SOURCE accepts PR's(pull requests) from **newbies** only, this is to help **newbies** get familiar with contribution processes.
+
+Issues can be submitted by anyone, seasoned developers or newbies.
+
+# Getting Started
+
+1.  Fork the project on Github.
```

Comment:
The first step would benefit with the use of an image to show how to do it.

The rest of the steps assume that the contributor knows how to use git and is comfortable with the terminal. A section introducing the prerequisites can be added prior to this. The intro can point to existing resources for learning git like https//:try.github.io and the Pro Git book.